### DISCUSSION TITLE
Is there a better way to redirect table record edit

### DISCUSSION THREAD

**Discussion by aurawindsurfing** [filamentphp/filament#8512] [Help]
Hi,

Hey @danharrin I'm currently redirecting table record using the following:

```
Tables\Actions\EditAction::make()->url(fn($record) => route('filament.hub.resources.client.edit', $record)),
```

But I think I have seen a more neat way to to this with use of getRecord, can anyone point me to the right place, please?

**Comment by leandrocfe** [comment]
```php
->url(fn ($record): string => ClientResource::getUrl('edit', ['record' => $record]))
```
But why would you like to do this?

Please don't tag Dan. If he has a tip, he will help you

**Comment by aurawindsurfing** [comment]
@leandrocfe thanks for the replay and sorry for the tag.

I use it to customise the edit action on the resource.

Which is described here: https://filamentphp.com/docs/3.x/tables/advanced#record-urls-clickable-rows

But was just wondering if there is a neat helper to deal with it.

**Reply by leandrocfe** [reply]
Hum, I don't know, just my suggestion using getUrl
